cypress/e2e/code-of-conduct.cy.js

```diff
@@ -0,0 +1,102 @@
+describe('Tests Code of Conduct page', () => {
+    beforeEach(() => {
+      cy.visit('https://womencodingcommunity.com/mentorship-code-of-conduct');
```

Comment:
I think the domain should be environment variable or at least configuration that we can pre-define as `https://womencodingcommunity.com` and easily override, because if I am running locally this will be different domain, for example `http://localhost:4000/`. This will make easier when you implement the logic to run locally.